jovo-integrations/jovo-db-dynamodb/src/DynamoDb.ts

```diff
@@ -127,11 +133,16 @@ export class DynamoDb implements Db {
     const getDataMapParams: DocumentClient.GetItemInput = {
       ConsistentRead: true,
       Key: {
-        [this.config.primaryKeyColumn!]: primaryKey,
+        [this.config.primaryKeyColumn!]: `${this.config.primaryKeyPrefix!}${primaryKey}`,
```

Comment:
The default scenario (without the prefix) would result in `undefined${primaryKey}` here. Is that correct? I would use '' as default value for `this.config.primaryKeyPrefix`.